ldes-consumer/kgap/spawn_instances.py

```
@@ -9,71 +9,131 @@
+
+
+# === Docker container management functions ===
+def docker_container_name(feedname: str) -> str:
+    """Generate the Docker container name for a given feed."""
+    return f"{pfx}-{feedname}"
+
+
+@contextmanager
+def check_docker_container_running(
+    feedname: str,
+    feed: dict
```

Ruff (0.14.8)

Unused function argument: `feed`

(ARG001)